### DISCUSSION TITLE
Is it possible to get the Gantt chart of multiple projects?

### DISCUSSION THREAD

**Discussion by denisvitez** [Leantime/leantime#1332] [Q&A] [answered]
Hi,
I have been trying the Leantime for the past day and I really like it. However, I'm wondering if there is any option to see the Gantt chart of all active projects together and not just the current selected project?

I guess one alternative would be to have one project and then use Milestones as my project. But this would lose some of the functionality that the app offers.

I'm hopeful I just somewhere missed the option to see more projects on the chart :)

**Comment by marcelfolaron** [comment, marked as answer]
Hi Denis,

That is currently not a feature, but something we are working on. More than
likely it will be released as a plugin.
